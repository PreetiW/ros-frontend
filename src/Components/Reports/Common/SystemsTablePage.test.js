import { test } from '@jest/globals';

test.todo('Systems Table Page Component');
