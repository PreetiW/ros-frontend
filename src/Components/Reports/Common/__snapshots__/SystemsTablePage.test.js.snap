// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SystemsTablePage Component should matches snapshot for systems table page and renders correctly 1`] = `
<DocumentFragment>
  <view>
    <text
      style="font-size: 9px; font-weight: 700; color: rgb(201, 25, 11);"
    />
    <view
      style="display: flex; flex-direction: row; justify-content: space-between;"
    >
      <view
        style="flex: 1;"
      >
        <view>
          <view
            style="display: flex; flex-direction: row; justify-content: space-between; padding-left: 0.5rem; padding-bottom: 0.5rem; padding-top: 0.5rem;"
          >
            <text>
              Name
            </text>
            <text>
              Group
            </text>
            <text>
              OS
            </text>
            <text>
              CPU utilization
            </text>
            <text>
              Memory utilization
            </text>
            <text>
              I/O utilization
            </text>
            <text>
              Suggestions
            </text>
            <text>
              State
            </text>
            <text>
              Last reported
            </text>
          </view>
          <view
            style="border-top-style: solid; border-top-color: #8a8d90; border-top-width: 1px; border-bottom-style: solid; border-bottom-color: #8a8d90; border-bottom-width: 1px;"
          >
            <view
              style="display: flex; flex-direction: row; justify-content: space-between; padding-left: 0.5rem; padding-bottom: 0.5rem; padding-top: 0.5rem;"
            >
              <text>
                iqe-ros-bb519f81
              </text>
              <text>
                RHEL 8.4
              </text>
              <text>
                90%
              </text>
              <text>
                80%
              </text>
              <text>
                0.314
              </text>
              <text>
                2
              </text>
              <text>
                Undersized
              </text>
              <text>
                2022-03-29
              </text>
            </view>
            <view
              style="display: flex; flex-direction: row; justify-content: space-between; padding-left: 0.5rem; padding-bottom: 0.5rem; padding-top: 0.5rem; background-color: rgb(240, 240, 240);"
            >
              <text>
                iqe-ros-bb519f81
              </text>
              <text>
                RHEL 8.4
              </text>
              <text>
                90%
              </text>
              <text>
                80%
              </text>
              <text>
                0.314
              </text>
              <text>
                2
              </text>
              <text>
                Under pressure
              </text>
              <text>
                2022-03-30
              </text>
            </view>
            <view
              style="display: flex; flex-direction: row; justify-content: space-between; padding-left: 0.5rem; padding-bottom: 0.5rem; padding-top: 0.5rem;"
            >
              <text>
                iqe-ros-bb519f81
              </text>
              <text>
                RHEL 8.4
              </text>
              <text>
                N/A
              </text>
              <text>
                N/A
              </text>
              <text>
                N/A
              </text>
              <text>
                2
              </text>
              <text>
                Waiting for data
              </text>
              <text>
                2022-03-31
              </text>
            </view>
          </view>
        </view>
      </view>
    </view>
  </view>
</DocumentFragment>
`;
